### PR TITLE
refactor: image full token

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   -        |
| 🇨🇳 Chinese |    -       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/image/style/index.tsx [hunk 1]
```diff
@@ -107,9 +108,9 @@ export const genPreviewSwitchStyle = (token: ImageToken): CSSObject => {
     display: 'flex',
     alignItems: 'center',
     justifyContent: 'center',
-    width: token.imagePreviewSwitchSize,
-    height: token.imagePreviewSwitchSize,
-    marginTop: -token.imagePreviewSwitchSize / 2,
+    width: token.controlHeightLG,
```
Comment (zombieJ) [inline components/image/style/index.tsx:111]: `imagePreviewSwitchSize` 还是要的，但是是在 `imageToken` 里

Comment (MadCcc) [reply components/image/style/index.tsx]: done

File: components/image/style/index.tsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+import { modalMask } from '../../modal/style';
 import type { FullToken, GenerateStyle } from '../../_util/theme';
 import { genComponentStyleHook, mergeToken, resetComponent } from '../../_util/theme';
 
 export interface ComponentToken {
   imageBg: string;
-  imagePreviewOperationSize: number;
   imagePreviewOperationColor: string;
-  imagePreviewSwitchSize: number;
   zIndexImage: number;
```
Comment (zombieJ) [inline components/image/style/index.tsx:12]: 这个名字看起来有问题，`zIndexDropdown` 指的是下拉框。但是 Image 和 Modal 里其实是弹出框。这么看的话所有的 `zIndexXXX` 都应该叫做 `zIndexPopup`

File: components/image/style/index.tsx [hunk 3]
```diff
@@ -276,15 +258,15 @@ export default genComponentStyleHook(
...
 
-    return [genImageStyle(imageToken)];
+    return [genImageStyle(imageToken), genImagePreviewStyle(imageToken)];
   },
   {
     imageBg: '#f5f5f5',
-    imagePreviewOperationSize: 18,
     imagePreviewOperationColor: new TinyColor({ r: 255, g: 255, b: 255, a: 0.85 }).toRgbString(),
-    imagePreviewSwitchSize: 44,
     zIndexImage: 1080,
```
Comment (zombieJ) [inline components/image/style/index.tsx:270]: 这个是基于标准 zIndexPopup 算出来的，不是 hardcode，参考一下 Select 的写法

File: components/image/style/index.tsx [hunk 4]
```diff
@@ -1,14 +1,14 @@
 // deps-lint-skip-all
 import type { CSSObject } from '@ant-design/cssinjs';
 import { TinyColor } from '@ctrl/tinycolor';
+import type { ModalToken } from '../../modal/style';
+import { modalMask } from '../../modal/style';
 import type { FullToken, GenerateStyle } from '../../_util/theme';
 import { genComponentStyleHook, mergeToken, resetComponent } from '../../_util/theme';
 
 export interface ComponentToken {
   imageBg: string;
```
Comment (zombieJ) [inline components/image/style/index.tsx:10]: 使用通用 token，不归 Image 管

File: components/image/style/index.tsx [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
-  imagePreviewOperationSize: number;
   imagePreviewOperationColor: string;
```
Comment (zombieJ) [inline components/image/style/index.tsx:11]: 找一下有没有复用的颜色，不做成 Image 独有 Token

**Review by zombieJ — APPROVED** [review]

**Pull request merged**